### PR TITLE
Add example usage of TerraformTaskV3 and fix typos

### PR DESCRIPTION
**Pull request by hikilaka**
I have added a few example usages of the TerraformTaskV3 to the README for this task. Additionally, I have corrected a few typos in the same README.

### CODE REVIEW

**Comment by hikilaka** [comment]
I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.

**Comment by mericstam** [comment]
looks good! I just want to verify one thing regarding service connections before I review and approve it

File: Tasks/TerraformTask/TerraformTaskV3/README.md [hunk 1]
```diff
@@ -98,3 +98,38 @@ When selecting custom you can use any command that is supported natively by terr
...
+    backendAzureRmKey: 'state.tfstate'
+
+- task: TerraformTaskV3@3
+  name: terraformPlan
+  displayName: Create Terraform Plan
+  inputs:
+    provider: 'azurerm'
+    command: 'plan'
+    commandOptions: '-out main.tfplan'
+    environmentServiceNameAzureRM: 'your-service-connection'
```
Comment (mericstam) [inline Tasks/TerraformTask/TerraformTaskV3/README.md:124]: sam here.
```suggestion
    environmentServiceNameAzureRM: 'your-environment-service-connection'
```

File: Tasks/TerraformTask/TerraformTaskV3/README.md [hunk 2]
```diff
@@ -98,3 +98,38 @@ When selecting custom you can use any command that is supported natively by terr
...
+
+# Only runs if the 'terraformPlan' task has detected changes the in state. 
+- task: TerraformTaskV3@3
+  displayName: Apply Terraform Plan
+  condition: eq(variables['terraformPlan.changesPresent'], 'true')
+  inputs:
+    provider: 'azurerm'
+    command: 'apply'
+    commandOptions: 'main.tfplan'
+    environmentServiceNameAzureRM: 'your-service-connection'
```
Comment (mericstam) [inline Tasks/TerraformTask/TerraformTaskV3/README.md:134]: And here as well
```suggestion
        environmentServiceNameAzureRM: 'your-environment-service-connection'
```

**Review by mericstam — CHANGES_REQUESTED** [review]
LGTM but consider changes

**Review by mericstam — APPROVED** [review]

**Pull request merged**